src/lib/Behat/PageElement/AdminList.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\EzPlatformAdminUi\Behat\PageElement;
+
+use Behat\Mink\Exception\ElementNotFoundException;
+use EzSystems\EzPlatformAdminUi\Behat\Helper\UtilityContext;
+
+class AdminList extends Element
+{
+    protected $fields = ['plusButton' => '.ez-icon-create',
+                        'trashButton' => '.ez-icon-trash',
+                        'editButton' => '.ez-icon-edit',
+                        'listHeader' => '.ez-table-header__headline',
+                        'listElementLink' => '.ez-checkbox-cell+ td a', ];
+    /** @var string Name by which Element is recognised */
+    public const ELEMENT_NAME = 'Admin List';
+    protected $listHeader;
+
+    public function __construct(UtilityContext $context, string $listHeader)
+    {
+        parent::__construct($context);
+        $this->listHeader = $listHeader;
+    }
+
+    private function assertProperList(): void
+    {
+        if (strcmp($this->listHeader, $this->context->findElement($this->fields['listHeader'], 5)->getText()) !== 0) {
+            throw new ElementNotFoundException($this->context->getSession(), 'table header', $this->fields['listHeader']);
+        }
+    }
+
+    public function clickPlusButton(): void
+    {
+        $this->context->findElement($this->fields['plusButton'])->click();
+    }
+
+    public function clickTrashButton(): void
+    {
+        $this->context->findElement($this->fields['trashButton'])->click();
+    }
+
+    public function selectListElement(string $name): void
+    {
+    }
+
+    public function clickListElement(string $name): void
+    {
+        $this->context->getElementByText($name, $this->fields['listElementLink'])->click();
+    }
+
+    public function checkListElement(string $name): bool
+    {
+        return $this->context->getElementByText($name, $this->fields['listElementLink']) !== null;
+    }
+
+    public function verifyVisibility(): void
+    {
+        $this->context->checkVisibilityByClass($this->fields['plusButton']);
```

Comment:
It won't work: checkVisibility returns bool (does not throw). 
We can use:
`Assert:assertTrue(checkVisibilityByClass());`
or (assuming element exists):
`Assert:assertTrue(findElement(selector)->isVisible());`

I guess the name is misleading, we might want to change that.